### ISSUE TITLE
benchmark compilation fail on MacOSX

### ISSUE THREAD

**Issue by cedrict** [geodynamics/aspect#469]
when executing cmake . in various benchmakr folders, it always returns the following message :
```
-- Using the deal.II-8.2.1 installation found at /Applications/deal.II.app/Contents/Resources
-- Include macro /Applications/deal.II.app/Contents/Resources/cmake/macros/macro_deal_ii_initialize_cached_variables.cmake
-- Include macro /Applications/deal.II.app/Contents/Resources/cmake/macros/macro_deal_ii_invoke_autopilot.cmake
-- Include macro /Applications/deal.II.app/Contents/Resources/cmake/macros/macro_deal_ii_setup_target.cmake
-- Setting up plugin:
--   name <solkz>
--   using ASPECT_DIR=
--   in Release mode
-- Configuring done
CMake Warning (dev):
  Policy CMP0042 is not set: MACOSX_RPATH is enabled by default.  Run "cmake
  --help-policy CMP0042" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

  MACOSX_RPATH is not specified for the following targets:

   solkz

This warning is for project developers.  Use -Wno-dev to suppress it.

-- Generating done
-- Build files have been written to: /Users/geogarfield/work/ASPECT/aspect_20may2015/aspect/benchmark/solkz
```
When executing make, it returns:

```
Linking CXX shared library libsolkz.dylib
Undefined symbols for architecture x86_64:
  "aspect::Postprocess::Manager<2>::register_postprocessor(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, void (*)(dealii::ParameterHandler&), aspect::Postprocess::Interface<2>* (*)())", referenced from:
      __GLOBAL__I_a in solkz.cc.o
  "aspect::Postprocess::Manager<3>::register_postprocessor(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, void (*)(dealii::ParameterHandler&), aspect::Postprocess::Interface<3>* (*)())", referenced from:
      __GLOBAL__I_a in solkz.cc.o
  "aspect::Postprocess::Interface<2>::initialize()", referenced from:
      vtable for aspect::InclusionBenchmark::SolKzPostprocessor<2> in solkz.cc.o
      vtable for aspect::Postprocess::Interface<2> in solkz.cc.o
  "aspect::Postprocess::Interface<2>::parse_parameters(dealii::ParameterHandler&)", referenced from:
      vtable for aspect::InclusionBenchmark::SolKzPostprocessor<2> in solkz.cc.o
      vtable for aspect::Postprocess::Interface<2> in solkz.cc.o
... ... ... 
```

**Comment by tjhei** [comment]
do the tests work for you? (run "make test")

**Comment by cedrict** [comment]
nope:

bash-3.2$ make test
Running tests...
Test project /Users/geogarfield/work/ASPECT/aspect_20may2015/aspect
    Start 1: quick_mpi
1/1 Test #1: quick_mpi ........................***Failed    6.45 sec

0% tests passed, 1 tests failed out of 1

Total Test time (real) =   6.45 sec

The following tests FAILED:
  1 - quick_mpi (Failed)


--------------------------------------------------------------
Completed running the minimal testsuite. Please check the test
results above. To run the complete testsuite, reconfigure your
project with the option ASPECT_RUN_ALL_TESTS=ON or by running
    make setup_tests
Be aware that installing the  numdiff' tools is strongly'
recommended before running the full testsuite. Even then, the
output is likely different on your machine due to configuration
differences, so do not expect all tests to succeed on your
machine.
--------------------------------------------------------------
Errors while running CTest
make: *** [test] Error 8

**Comment by tjhei** [comment]
Can you please run "ctest -V" so we can see if it is the same problem?

**Comment by tjhei** [comment]
> --   using ASPECT_DIR=
weird, you are finding ASPECT, but there is no ASPECT directory set.

**Comment by cedrict** [comment]
I still get the same error and the make crashes:


-- Using the deal.II-8.2.1 installation found at /Applications/deal.II.app/Contents/Resources
-- Include macro /Applications/deal.II.app/Contents/Resources/cmake/macros/macro_deal_ii_initialize_cached_variables.cmake
-- Include macro /Applications/deal.II.app/Contents/Resources/cmake/macros/macro_deal_ii_invoke_autopilot.cmake
-- Include macro /Applications/deal.II.app/Contents/Resources/cmake/macros/macro_deal_ii_setup_target.cmake
-- Setting up plugin:
--   name <solkz>
--   using ASPECT_DIR=/Users/geogarfield/work/ASPECT/aspect_20may2015/aspect
--   in Release mode
-- Configuring done
CMake Warning (dev):
  Policy CMP0042 is not set: MACOSX_RPATH is enabled by default.  Run "cmake
  --help-policy CMP0042" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

  MACOSX_RPATH is not specified for the following targets:

   solkz

This warning is for project developers.  Use -Wno-dev to suppress it.

-- Generating done
-- Build files have been written to: /Users/geogarfield/work/ASPECT/aspect_20may2015/aspect/benchmark/solkz

**Comment by tjhei** [comment]
Okay, figured it out. I am preparing a PR...

**Comment by tjhei** [comment]
addressed by #480
